server/server.go

```diff
@@ -108,5 +108,5 @@ func HandleRequests() {
 	http.HandleFunc("/api/collect/hosts", hostTelemetryHandler(influxDBAdapter, postgresAdapter))
 
 	log.Infof("Starting Trento telemetry server...")
-	log.Fatal(http.ListenAndServe(":10000", nil))
+	log.Fatal(http.ListenAndServe(":80", nil))
```

Comment:
Could we switch to `8080` and make this configurable via env variable? `:80` means that I have to be root to start the binary and make it bind successfully, a thing that maybe I wouldn't desire basically for development purposes or just because I want to run the binary under a different user :sweat_smile:

Reply:
yes, we can add a CLI flag and make it configurable, for now I need to finalize whatever is the thing that is going to end up in production, and I don't want to use port 10000 in production because that's not what I would expect from a plain http service behind a load balancer.

Reply:
I understand, what about `:8080`? I still wouldn't use `:80` :sweat_smile: